### PR TITLE
[docs] update README Runner section

### PR DESCRIPTION
**Pull request by mxm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,15 @@ Have ideas for new SDKs or DSLs? See the [Jira](https://issues.apache.org/jira/b
 
 ### Runners
 
-Beam supports executing programs on multiple distributed processing backends. After the Beam project's initial bootstrapping completes, it will include:
-  1. The `DirectPipelineRunner` runs the pipeline on your local machine.
-  2. The `DataflowPipelineRunner` submits the pipeline to the [Google Cloud Dataflow](http://cloud.google.com/dataflow/).
-  3. The `SparkPipelineRunner` runs the pipeline on an Apache Spark cluster. See the code that will be donated at [cloudera/spark-dataflow](https://github.com/cloudera/spark-dataflow).
-  4. The `FlinkPipelineRunner` runs the pipeline on an Apache Flink cluster. See the code that will be donated at [dataArtisans/flink-dataflow](https://github.com/dataArtisans/flink-dataflow).
+Beam supports executing programs on multiple distributed processing backends (runners). It currently includes the following Runners:
```
Comment (davorbonaci) [inline README.md]: I think of "distributed processing backends" as being different from "runners". A runner is the component that takes a Beam pipeline and submits it to the "processing backend", but doesn't include it. E.g., FlinkPipelineRunner vs. Flink backend.

Suggested: "Beam supports executing programs on multiple distributed processing backends. Currently, it supports the following:"

File: README.md [hunk 2]
```diff
@@ -44,11 +44,15 @@ Have ideas for new SDKs or DSLs? See the [Jira](https://issues.apache.org/jira/b
...
+
+- The `DirectPipelineRunner` runs the pipeline on your local machine.
+- The `DataflowPipelineRunner` submits the pipeline to the [Google Cloud Dataflow](http://cloud.google.com/dataflow/).
+- The `FlinkPipelineRunner` runs the pipeline on an Apache Flink cluster. The code has been donated from [dataArtisans/flink-dataflow](https://github.com/dataArtisans/flink-dataflow) and is now part of Beam.
+
+After the Beam project's initial bootstrapping completes, it will include:
+- The `SparkPipelineRunner` runs the pipeline on an Apache Spark cluster. See the code that will be donated at [cloudera/spark-dataflow](https://github.com/cloudera/spark-dataflow).
+- The `GearpumpRunner` runs the pipeline on a Gearpump: [Gearpump](https://github.com/gearpump/gearpump)
```
Comment (davorbonaci) [inline README.md]: Let's not "promise" this yet.

**Comment by davorbonaci** [comment]
CC: @francesperry 

Nice!

File: README.md (hunk 2 above)
Comment (mxm) [inline README.md]: Are you referring to the Gearpump runner?

File: README.md (hunk 1 above)
Comment (mxm) [inline README.md]: Yes absolutely. Would be nice to explain the notion of a Runner as well. How about changing it to "Beam supports executing programs on multiple distributed processing backends through Runners. Currently, the following Runners are supported:".

Comment (davorbonaci) [inline README.md]: Let's use `PipelineRunner` instead of Runner.

File: README.md (hunk 2 above)
Comment (davorbonaci) [inline README.md]: Yup.

**Comment by davorbonaci** [comment]
LGTM.

**Comment by mxm** [comment]
Thanks. I've updated the pull request with your suggestions. Will merge this later on.

**Pull request closed without merging**